test/WebJobs.Extensions.RabbitMQ.Tests/RabbitMQEndToEndTests.cs

```diff
@@ -0,0 +1,109 @@
+﻿//using System;
+//using System.Collections.Generic;
+//using System.Text;
+//using System.Linq;
+//using System.Threading.Tasks;
+//using WebJobs.Extensions.RabbitMQ.EndToEnd;
+//using Microsoft.Extensions.Configuration;
+//using Microsoft.Azure.WebJobs.Extensions.Tests.Common;
+//using Microsoft.Extensions.DependencyInjection;
+//using Microsoft.Extensions.Hosting;
+//using Microsoft.Extensions.Logging;
+//using RabbitMQ.Client;
+//using RabbitMQ.Client.Events;
+//using Xunit;
+
+//namespace Microsoft.Azure.WebJobs.Extensions.RabbitMQ.Tests
+//{
+//    public class RabbitMQEndToEndTests
+//    {        
+//        private const string HostName = "localhost";
+//        private const string QueueName = "queue";
+//        private const string Message = "Hello";
+//        private readonly TestLoggerProvider _loggerProvider = new TestLoggerProvider();
+
+//        [Fact]
+//        public async Task RabbitMQEndToEnd()

```

Comment:
@brettsam / @fabiocav - Are there any plan to move E2E tests to a central repo? These tests need to use common code that is part of extensions repo

Reply:
Yes there are a bunch of test helpers that really should be centralized, especially now that bindings are all moving to their own repos. Let's chat about that separately and see what we can do.